docs/docs/lib/build-your-own.mdx

```diff
@@ -966,6 +970,11 @@ channel.addEventListener("features", (event) => {
 })
 ```
 
+Some important things to note:
+
+- The connection will close after 60 seconds. It is expected that the SDK implements reconnect logic.
```

Comment:
Not sure I'd mention this, just that connections are not permanent and the client is responsible for reconnecting. We're probably going to increase to ~5 mins, but they should also handle client-side disconnections (poor data connection, for example)

Reply:
I've updated to remove the specifics around disconnection.